bot/util/utilities.py

```diff
@@ -150,6 +153,18 @@ async def cembed(self, ctx, channel: str, *, json_string: str):
         except TypeError:
             await ctx.send("**__Error:__** Error creating embed. Please check your parameters.")
 
+    @commands.command(name="echo")
+    async def echo(self, ctx, channel: typing.Optional[discord.TextChannel], *, text: str):
+        """Lets the bot post a simple message to the mentioned channel (or the current channel if none is mentioned).
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            channel (Optional[str]): The channel to post the embed to.
+            text (str): The text to be echoed (might begin with the channel mention)
```

Comment:
`might begin with the channel mention`? - Why's that? Do you mean every time one is mentioned at all?
It would be better to remove the mention in my opinion.

Dot missing.